app/views/provider_interface/interviews/new.html.erb

```diff
@@ -1 +1,34 @@
-<h1> New interview page </h1>
+<% content_for :browser_title, title_with_error_prefix("Set up an interview", nil) %>
+<% content_for :before_content, govuk_back_link_to %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @interview_form,
+          url: new_check_provider_interface_application_choice_interviews_path(@interview_form.application_choice.id),
+          method: :get do |f| %>
+
+      <%= f.govuk_error_summary %>
+
+      <span class="govuk-caption-l"><%= @interview_form.application_choice.application_form.full_name %></span>
+      <h1 class="govuk-heading-l">Set up an interview</h1>
+
+      <%= f.govuk_date_field :date, legend: { text: 'Date', size: 'm' }, hint: { text: 'For example, 31 3 2020' } %>
```

Comment:
There's a bug where this does not get re-populated with the submitted data when a validation error occurs. Anyone have any ideas why?

Reply:
Is it because the original `3i` `2i` and `1i` fields aren't populated on the form object, being transformed instead in `InterviewsController#interview_params`?